addons/stock_account/models/stock_move.py

```diff
@@ -212,9 +212,9 @@ def _create_dropshipped_svl(self, forced_quantity=None):
                 valued_quantity += valued_move_line.product_uom_id._compute_quantity(valued_move_line.quantity, move.product_id.uom_id)
             quantity = forced_quantity or valued_quantity
 
-            unit_cost = move._get_price_unit()
-            if move.product_id.cost_method == 'standard':
-                unit_cost = move.product_id.standard_price
+            unit_cost = move.product_id.standard_price
```

Comment:
this order makes more sense 
but is this part of the fix?

Reply:
It's not directly related to the fix, however if we re-order it this way in the future if there are problems with this code, we will accurately see the issue for all affected picking types and cost methods rather than it, in some cases, seeming as though only certain picking types and cost methods are broken.

Reply:
@ethanrobv I would not change this part. It doesn't fix anything to me